addon-test-support/-private/is-focusable.js

```diff
@@ -0,0 +1,10 @@
+export default function isFocusable(el) {
+  let focusableTags = ['INPUT', 'BUTTON', 'LINK', 'SELECT', 'A', 'TEXTAREA'];
+  let { tagName, type } = el;
+
+  if (type === 'hidden') {
+    return false;
+  }
+
+  return focusableTags.includes(tagName) || el.contentEditable;
```

Comment:
hm, babel is not going to polyfill it correctly? I thought its doing it by itself. If not, I'll change back to indexOf